src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -32,28 +32,22 @@
 
 DATA_SAVE_ID  = wx.NewId()
 
-
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var to exclude subplots
+        self.toolitems = tuple(el for el in NavToolbar.toolitems\
+                if el[0] != 'Subplots')
         NavToolbar.__init__(self, canvas)
-        # Get rid of the configure subplots button
-        if hasattr(self, 'wx_ids'):
-            self.DeleteTool(self.wx_ids['Subplots'])
-        else:
-            self.DeleteToolByPos(6)
         # Add new buttons
         self.AddSimpleTool(wx.ID_PRINT,
                wx.ArtProvider.GetBitmap(wx.ART_PRINT, wx.ART_TOOLBAR),
                'Print', 'print graph')
         self.AddSimpleTool(DATA_SAVE_ID,
-               _load_bitmap('stock_save_as.xpm'),
+               _load_bitmap('filesave.png'),
```

Comment:
thanks, symlink solves the problem and here is the result of testing whole ``pdfgui`` with the grayscale icon in **MPL2.0**

![image](https://cloud.githubusercontent.com/assets/15218640/26741537/bd6bfdb0-47a7-11e7-8440-4ddf683a1292.png)

and the same grayscale icon in **MPL1.5.3**

![image](https://cloud.githubusercontent.com/assets/15218640/26741770/bd029720-47a8-11e7-9d21-5c105f4d2be8.png)